microsoft-edge/webview2/get-started/hololens2.md

```diff
@@ -176,89 +176,31 @@ For more information, see [Welcome to the Mixed Reality Feature Tool](/windows/m
 
 
 <!-- ====================================================================== -->
-## Step 6 - Configure WebView2 settings and properties
-
-While the prefab exposes just a single property, there is more functionality exposed on the `WebView` script. Let's take a look are exposing some of this functionality in our project. First, we recommend reviewing the `WebView` script to get an idea of what's available. There is further functionality available in the underlying WebView2, but this will give you a good idea of the basics.
-
-To view the `WebView` script, in the **Inspector** window, double-click the `WebView` script. This opens the script in your default code editor.
-
-Now that you've reviewed the `WebView` code, let's extend the functionality of our sample. To do that, we'll create a new script that derives from `WebView`.
-
-1. In the **Inspector** window for WebView, click **Add Component**, select **New script**, type **WebViewBrowser**, and then click **Create and add**. Your new component is added to the **Inspector** window.
-
-1. In the **Inspector** window, double-click the `WebViewBrowser` script, to edit the script.
-
-1. Replace the contents of that file with the following code:
-
-   ```C#
-   using Microsoft.MixedReality.WebView;
-   using UnityEngine.UI;
-   using UnityEngine;
-   using TMPro;
-   using System;
-
-   public class WebViewBrowser: MonoBehaviour
-   {
-      // Declare UI elements: Back button, Go button, and URL input field
-      public Button BackButton;
-      public Button GoButton;
-      public TMP_InputField URLField;
+## Step 6 - Test the project
 
-      private void Start()
-      {
-         // Get the WebView component attached to the game object
-         var webViewComponent = gameObject.GetComponent<WebView>();
-         webViewComponent.GetWebViewWhenReady((IWebView webView) =>
-         {
-               // If the WebView supports browser history, enable the Back button
-               if (webView is IWithBrowserHistory history)
-               {
-                  // Add an event listener for the Back button to navigate back in history
-                  BackButton.onClick.AddListener(() => history.GoBack());
-
-                  // Update the Back button's enabled state based on whether there's any history to go back to
-                  history.CanGoBackUpdated += CanGoBack;
-               }
-
-               // Add an event listener for the Go button to load the URL that was entered in the input field
-               GoButton.onClick.AddListener(() => webView.Load(new Uri(URLField.text)));
+Now that you have added the WebView plugin to the scene, it would be a good time to test your project.   You can test the project directly in the editor, or you can follow these steps to [Build and deploy to the HoloLens](/windows/mixed-reality/develop/unity/build-and-deploy-to-hololens)
```

Comment:
```suggestion
Now that you have added the WebView plugin to the scene, it would be a good time to test your project.   You can test the project directly in the editor, or you can follow these steps to [Build and deploy to the HoloLens](/windows/mixed-reality/develop/unity/build-and-deploy-to-hololens).
```